Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -21,6 +21,12 @@ compatible with workspaces is described in the
 TCA reference (in the :ref:`description of the "ctrl" section <t3tca:ctrl>`
 and in the :ref:`description of the "versioningWS" property <t3tca:ctrl-reference-versioningws>`).
 
+You might want to turn the workspace off for certain tables. 
+You can do so in the :file:`AdditionalConfiguration.php`::
```

Comment:
No, this must happen in TCA overrides instead. Can you link to that?

Reply:
But this is also required in the AdditionalConfiguration.php . Does this not work?
You cannot write your own extension because of this.

Reply:
No, all TCA overrides *must* be done properly, the `AdditionalConfiguration.php` is not the place for this.

So yes, you *must* have at least one extension to define TCA overrides. Given that nowadays every TYPO3 instance has something like a site package, this is not a problem however.

Reply:
Is there already a documentation for a site package? Then I could make a reference link to it.

Reply:
For example: https://docs.typo3.org/m/typo3/tutorial-sitepackage/master/en-us/

The following is also related: https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ExtendingTca/StoringChanges/Index.html